packages/perseus-editor/src/components/graph-locked-figures/locked-function-settings.tsx

```diff
@@ -200,6 +256,43 @@ const LockedFunctionSettings = (props: Props) => {
     );
 };
 
+type ItemProps = {
+    category: string;
+    example: string;
+    index: number;
+    pasteEquationFn: (property: string, newValue: string) => void;
+};
+
+const ExampleItem = (props: ItemProps): React.ReactElement => {
+    const {category, example, index, pasteEquationFn} = props;
+    const exampleId = useId();
+
+    return (
+        <li key={`${category}-${index}`} className={css(styles.exampleRow)}>
+            <IconButton
+                icon={autoPasteIcon}
+                aria-label="paste example"
+                aria-describedby={exampleId}
+                onClick={() => pasteEquationFn("equation", example)}
+                size="medium"
```

Comment:
I think if you use `size="small"` here, it'll have the icons remain the same size, and the padding around them becomes smaller. This might be worth trying and seeing if the spacing looks better inline here? Let me know what you think.

Reply:
Yes, specifying "small" does keep the icon the same size. However, I'm going to keep it as is in order to keep the target area the current size (40px) as a best practice. If we get word from content creators that they prefer more space for the equation text, then we can adjust at that time.